python/src/space/core/storage.py

```diff
@@ -77,12 +83,24 @@ def __init__(self, location: str, metadata_file: str,
         self._physical_schema)
 
     self._primary_keys = set(self._metadata.schema.primary_keys)
+    self._current_branch = current_branch
+    self._max_snapshot_id = max(
+        [ref[1].snapshot_id for ref in self._metadata.refs.items()] +
+        [self._metadata.current_snapshot_id])
 
   @property
   def metadata(self) -> meta.StorageMetadata:
     """Return the storage metadata."""
     return self._metadata
 
+  @property
+  def current_branch(self) -> str:
+    """Return the current branch."""
+    if not self._current_branch:
```

Comment:
We can simply return self._current_branch, if self._current_branch is always populated (set to _MAIN_BRANCH).